templates/html/.filters/all.js

```diff
@@ -34,8 +34,16 @@ module.exports = ({ Nunjucks, Markdown, OpenAPISampler }) => {
 		if (!object) {
       throw new Error("object for containsTag was not provided?");
     }
+		if (!tagsToCheck) {
```

Comment:
I'm seeing weird indentation problems here and in some lines above. I recommend you to install [EditorConfig](https://editorconfig.org/) in your fav code editor. Installing [ESLint](https://eslint.org/) will also help.

Reply:
Have to double check, but thought I had it enabled. Fixed.